### DISCUSSION TITLE
How to use splat in MP engine?

### DISCUSSION THREAD

**Discussion by hzhben** [pelicanmapping/osgearth#1848] [Support Q&A] [answered]
How to use splat in MP engine? version 2.8

**Comment by gwaldron** [comment, marked as answer]
Sorry, but we no longer support version 2.8 or the deprecated/removed MP engine.

**Reply by hzhben** [reply]
Can you tell me the reason about osgearth_splat invalid when the engine driver is MP? And is it possible to modify the code to make MP support splat？
